pkg/filter/dt/filter_mysql_prepare.go

```diff
@@ -117,16 +124,18 @@ func (f *_mysqlFilter) processAfterPrepareDelete(ctx context.Context, conn *driv
 func (f *_mysqlFilter) processAfterPrepareInsert(ctx context.Context, conn *driver.BackendConnection,
 	result proto.Result, stmt *proto.Stmt, insertStmt *ast.InsertStmt) error {
 	has, xid := misc.HasXIDHint(insertStmt.TableHints)
+	newCtx, span := tracing.GetTraceSpan(ctx, "mysql_process_after_prepare_insert")
```

Comment:
I recommend not adding TraceContext for this location. instead, add inside of func `executor.AfterImage`:
```
newCtx, span := tracing.GetTraceSpan(ctx, "fetch_after_image")
```